packages/react-native-reanimated/src/errors.ts

```diff
@@ -18,11 +18,6 @@ const ReanimatedErrorConstructor: ReanimatedErrorConstructor =
     return errorInstance;
   } as ReanimatedErrorConstructor;
 
-export function registerReanimatedError() {
-  'worklet';
-  global.ReanimatedError = ReanimatedErrorConstructor;
-}
-
```

Comment:
Why did you remove it? We still want to register it in the global scope on Worklet runtimes, just not on the React runtime.

Reply:
Yeah, I can see that. I wanted to revert these changes but decided to keep them for now.
I don't like the idea of making `ReanimatedError` available globally in the worklet runtime but make it different for the react runtime. It totally makes no sense to me and is much more confusing than having `ReanimatedError` available globally.

Reply:
I would prefer to keep things consistent and either import the error in each file or use the global definition. If the first is not possible, I would just stick to the globally available `ReanimatedError`.